omnij-rpc/src/integ/groovy/foundation/omni/test/rpc/reorgs/SimpleSendReorgSpec.groovy

```diff
@@ -69,7 +69,7 @@ class SimpleSendReorgSpec extends BaseReorgSpec {
         when: "rolling back all blocks until before the initial funding"
         invalidateBlock(blockHashBeforeFunding)
         clearMemPool()
-        sleep(2000)  // Wait 2 seconds to avoid `ProcessNewBlock, block not accepted` (duplicate block)
+        delayAfterInvalidate()  // Sleep for a little while to avoid `ProcessNewBlock, block not accepted` (duplicate block)
```

Comment:
This is the only location that requires a 23 second delay to make sure that system time is more than the median.